packages/core/src/storage/migration/updates/0.3-0.3.1/index.ts

```diff
@@ -1,7 +1,10 @@
 import type { BaseAgent } from '../../../../agent/BaseAgent'
+import type { UpdateConfig } from '../../updates'
 
 import { migrateDidRecordToV0_3_1 } from './did'
 
-export async function updateV0_3ToV0_3_1<Agent extends BaseAgent>(agent: Agent): Promise<void> {
+// FIXME: optional update config parameter
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export async function updateV0_3ToV0_3_1<Agent extends BaseAgent>(agent: Agent, _config: UpdateConfig): Promise<void> {
```

Comment:
what is this fixme? I'm not sure i understand

Reply:
Me neither... that's the problem of make big refactors in the spare time :stuck_out_tongue_closed_eyes: